analysis/trigger_performance/PHL_trigger_report.Rmd

```diff
@@ -0,0 +1,349 @@
+#
+# # Data providers
+data_providers <- "OCHA Philippines, 510 Global"
+#
+# # Analysis
+# analysis <- "Historical data and analysis are available through IRI's Maproom [tool](http://iridl.ldeo.columbia.edu/fbfmaproom2/niger). The triggers were developed by IRI in close collaboration with DMN, WFP Niger, and the Centre for Humanitarian Data with valuable input from participating agencies and OCHA."
+#
+# Monitoring process
+monitoring_process <- "Between September and January, OCHA Philippines
+monitors typhoon forecasts from the Philippine Atmospheric, Geophysical
+and Astronomical Services Administration (PAGASA), the European Centre
+for Medium-Range Weather Forecasts (ECMWF), the Global Ensemble Forecast
+System (GEFS), and Weather Nerds (Private Initiative). If the typhoon
+is projected to impact areas in regions 5, 8 and 13 within 4-7 days, and exceed 178
+km/h 1-minute sustained wind speed, then the readiness trigger is reached, with
+either scenario 1 or 2 specified. 72 hours before landfall, the 510 damage
+assessment model is run. If any of the damage threshold numbers exceed
+those specified in the framework by the target probability
+(ranging from 50% probability that
+80,000 houses or more will be totally damaged, or 95% probability
+that 5,000 will be totally damaged), then the activation trigger is reached.
+Note that for a successful activation, both the readiness and activation
+trigger need to be reached."
+```
+
+```{r setup, include = FALSE, message = FALSE, warning = FALSE}
+knitr::opts_chunk$set(echo = FALSE) # do not print code by default
+knitr::opts_chunk$set(include = TRUE) # include chunk output by default
+knitr::opts_chunk$set(message = FALSE) # do not print messages by default
+knitr::opts_chunk$set(warning = FALSE) # do not print warnings by default
+
+options(scipen = 999) # turn off scientific notation
+options(encoding = "UTF-8") # set encoding to UTF-8 instead of ANSI
+```
+
+```{r libraries, include = FALSE, warning = FALSE}
+packages <- c('tidyverse', 'ggthemes', 'kableExtra', 'knitr', 'flextable')
+
+# install packages not yet installed
+installed_packages <- packages %in% rownames(installed.packages())
+
+if (any(installed_packages == FALSE)) {
+  install.packages(packages[!installed_packages])
+}
+
+# load libraries
+lapply(packages, library, character.only = TRUE)
+
+```
+
+
+```{r paths}
+data_folder_path <- paste0(Sys.getenv("AA_DATA_DIR"), "/public/exploration/", country_pcode3,"/trigger_performance/")
+perf_metrics_95_filename <- paste0(data_folder_path, country_pcode3, "_perf_metrics_table_ci_0.95.csv")
+perf_metrics_68_filename <- paste0(data_folder_path, country_pcode3, "_perf_metrics_table_ci_0.68.csv")
+learning_table_filename <- paste0(data_folder_path, country_pcode3, "_learning_table.csv")
+explanatory_ci_plot_path <- ("country_template/docs/explanatory_ci.png")
+```
+
+```{r functions}
+source("country_template/docs/plotAtvCI.R")
```

Comment:
Should the functional files be in a separate folder?

Reply:
What do you mean?